.github/workflows/Screenshots.yml

```diff
@@ -0,0 +1,191 @@
+name: Screenshot Tests
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+concurrency:
+  group: ${{ format('screenshot-{0}', github.event.pull_request.head.ref) }}
+  cancel-in-progress: true
+jobs:
+  record-base:
+    name: 📸 Record
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Set up JDK 11.
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Set up Android SDK.
+        uses: android-actions/setup-android@v2
+
+      - name: Set up Android SDK components.
+        run: |
+          # Uninstall NDK, because it's not used and it emits a warning:
+          # Warning: Observed package id 'ndk;21.4.7075529' in inconsistent location '/usr/local/lib/android/sdk/ndk-bundle' (Expected '/usr/local/lib/android/sdk/ndk/21.4.7075529')
+          echo sdkmanager --uninstall "ndk;21.4.7075529"
+          sdkmanager --uninstall "ndk;21.4.7075529"
+
+          # The following are already done by setup-android@v2 action:
+          # sdkmanager --install "cmdline-tools;latest"
+          # sdkmanager --install "tools"
+          # sdkmanager --install "platform-tools"
+
+          # Install Android API used to build the project.
+          echo sdkmanager --install "platforms;android-31"
+          sdkmanager --install "platforms;android-31"
+
+      - name: Checkout ${{ github.event.pull_request.base.ref }} branch in ${{ github.repository }} repository.
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ format('refs/heads/{0}', github.event.pull_request.base.ref) }}
+
+      - name: Record screenshots golden values for ${{ github.event.pull_request.base.ref }}.
+        id: gradle
+        run: >
+          ./gradlew
+          --no-daemon
+          --no-build-cache
+          --stacktrace
+          --continue
+          --scan
+          recordPaparazziDebug
+          -Pnet.twisterrob.build.screenshot-tests=true
+
+      - name: Publish "Gradle" result and Build Scan URL.
+        if: success() || failure()
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.repos.createCommitStatus({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              sha: context.sha,
+              state: "${{ steps.gradle.outputs.result-success }}" == "true" ? "success" : "failure",
+              context: "Screenshot Tests / Record / Gradle",
+              description: "${{ steps.gradle.outputs.result-text }}",
+              target_url: "${{ steps.gradle.outputs.build-scan-url }}"
+            });
```

Comment:
This didn't fail wihtout `steps.gradle` existing.

Reply:
JavaScript is unsafe, not much to do. Adding strictness with fromJSON as discovered in https://github.com/TWiStErRob/github-actions-test/pull/2

Reply:
https://github.com/TWiStErRob/net.twisterrob.sun/pull/91/commits/5cb04b923b227bcef0e0a826ffe3c9d45caaa859